src/intTest/resources/outbound_uat_data/amendment/app-j-4.edifact.dat

```diff
@@ -0,0 +1,14 @@
+UNB+UNOA:2+TES5+XX11+200610:1438+00000001'
```

Comment:
The "app-j" tests are taken directly from "Appendix J" of the spec. This one has been modified to test - I think only amending the house and street name. Could you change the filename maybe `house-street-names-only.edifact..`?

Reply:
ok, renamed